tests/E2E/test/campaigns/full/12_shop_parameters/1_enable_and_disable_shop.js

```diff
@@ -6,6 +6,9 @@ const {Menu} = require('../../../selectors/BO/menu.js');
 const welcomeScenarios = require('../../common_scenarios/welcome');
 let promise = Promise.resolve();
 
+const maintenance_messages = {en: 'We are currently disabled our shop and will be back really soon.',
```

Comment:
The english message is wrong 😅 

So the day we fix it in the Core, this test will fail :trollface:

Reply:
Fixed :100: even if it's not really relevant since we overwrite the message before checking it in the FO :trollface: